comparators/invalid_tag_modification.js

```diff
@@ -22,10 +22,14 @@ function invalidTagModification(newVersion, oldVersion, callback) {
   var result = {};
```

Comment:
@bkowshik 👋 ,instead of returning {} for a false result, we are now following convention to send false value itself. Ref commit: https://github.com/mapbox/osm-compare/commit/756c5cb05006cecba94508c89003813b07e28125